src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -103,13 +103,14 @@ static void usage()
             << "  --device <device path>  Specify nbd device path (/dev/nbd{num})\n"
             << "  --exclusive             Forbid writes by other clients\n"
             << "  --max_part <limit>      Override for module param max_part\n"
-            << "  --nbds_max <limit>      Override for module param nbds_max\n"
+            << "  --nbds_max <limit>      Override for module param nbds_max, will be ignored if --legacy is absent\n"
             << "  --quiesce               Use quiesce callbacks\n"
             << "  --quiesce-hook <path>   Specify quiesce hook path\n"
             << "                          (default: " << Config().quiesce_hook << ")\n"
             << "  --read-only             Map read-only\n"
             << "  --timeout <seconds>     Set nbd request timeout\n"
-            << "  --try-netlink           Use the nbd netlink interface\n"
+            << "  --legacy                Use the nbd ioctl interface for old kernels\n"
+            << "  --try-netlink           Deprecated. As default and will be remove later\n"
```

Comment:
```suggestion
            << "  --try-netlink           Use the nbd netlink interface (default, deprecated)\n"
```

Reply:
May be just remove `--try-netlink` from `usage` output? We don't want users who don't know about it to learn about it now. I think we only want the third party software that already uses it does not break.

Reply:
Cool, will remove it.